recipe/meta.yaml

```diff
@@ -55,4 +54,5 @@ extra:
     - andrewannex
     - jessemapel
     - scsides
-    - kberryUSGS
+    - kberry
```

Comment:
If you'd like to leave me on this list to periodically merge PRs (which is okay with me!), my new Github username is `krlberry`, not `kberry`.